src/segment-loader.js

```diff
@@ -1406,11 +1409,8 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     this.logger_(`checkBuffer_ returning ${segmentInfo.uri}`, {
       segmentInfo,
-      playlist,
       currentMediaIndex,
-      nextMediaIndex,
-      startOfSegment,
-      isSyncRequest
```

Comment:
These can actually be valuable to contain, in particular `startOfSegment`, which has been a source of various bugs we've encountered.

Reply:
These are all on segmentInfo, which is my reasoning behind removing them. But we could keep them as they keep things a bit more relevant to the checkBuffer log.

Reply:
I don't think these get logged except for here. Maybe worth adding to `segmentInfoString`?

Reply:
I have the next pull request in the series (where I refactor our segment selection logic) that adds selection to `segmentInfoString`. I think we should wait until that pull request. I will revert this logging change here for now.